addons/project/views/project_views.xml

```diff
@@ -360,14 +360,6 @@
                                 <field name="last_update_status" readonly="1" widget="status_with_color" options="{'color_field': 'last_update_color', 'no_quick_edit': 1}"/>
                             </div>
                         </button>
-                        <button class="oe_stat_button" type="object" name="action_view_analytic_account_entries" icon="fa-usd" attrs="{'invisible': [('analytic_account_id', '=', False)]}" groups="analytic.group_analytic_accounting">
```

Comment:
See with @IT-Ideas , I'm not sure if we can remove the button in stable even if it is duplicated. 🤔
For me, I think it is good and it should not break something. 😇

Reply:
Indeed as it is duplicated and as they have the same attributes, we can assume that xpath will not break.